lighthouse-core/gather/driver.js

```diff
@@ -464,8 +464,6 @@ class Driver {
         });
     }
 
-    log.verbose('Driver', `Installing longtask listener for CPU idle.`);
-    this.evaluateScriptOnLoad(`(${installPerformanceObserver.toString()})()`);
```

Comment:
Right, but now `waitForCpuIdle` can just be chained after `waitForNetworkIdle` rather than initialized at the same time and then marked as resolvable in the chain.